### PR TITLE
Update css folder structure

### PR DESCRIPTION
**Pull request by brad-decker**
1. rename `variables` to `design-system` to allow a combination of placeholder selectors (e.g `%shared-styles`), mixins (e.g `@mixin example`) and all other scss goodies alongside variables. 
2. moved breakpoint variables from the variables file in itcss to our new design system folder, I believe these will need to be updated but they are used enough to start bringing them in.
3. moved deprecated colors into a temporary file in design-system. This was done to allow for proper import orders while maintaining the structure and comments of the `css/index.scss` file.
4. Moved `generic/index.scss` to `css/base-styles.scss` because it actually has quite a bit of important styling in it and its previous location didn't relate to its significance. 
5. Moved `generic/reset.scss` to `css/reset.scss` for similar reasons
6. Import `generic/reset.scss` from `css/index.scss` instead of `css/base-styles.scss`
7. trimmed some unused styles in base-styles: I did a global search for the class names without a '.' and came up with only css files mentioned. as you review make sure I didn't remove anything critical. 

Depends on:
- ~#9018~

### CODE REVIEW

**Comment by brad-decker** [comment]
[Update fonts](https://trello.com/c/giqTMDWN/59-update-fonts)

File: ui/app/css/index.scss [hunk 1]
```diff
@@ -4,7 +4,9 @@
   They are included first because they will be used to replace bad variable names in itcss
   prior to it being fully removed from the system.
 */
-@import './variables/index';
+@import './design-system/index';
+@import './reset.scss';
```
Comment (Gudahtt) [inline ui/app/css/index.scss:8]: Shouldn't the CSS reset come first, before anything else? It _really_ shouldn't be overriding any of our styles, and putting it first is the best way to guarantee that.

Comment (brad-decker) [reply ui/app/css/index.scss:8]: 😅 you're absolutely right, i've no idea why it isn't the first thing.

Comment (brad-decker) [reply ui/app/css/index.scss:7]: resolved. Thanks @Gudahtt

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Pull request merged**